pkg/utils/data_generator_test.go

```diff
@@ -0,0 +1,69 @@
+package utils
+
+import (
+	"math"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestInvariants(t *testing.T) {
+	assert := assert.New(t)
+
+	const rows = 20
```

Comment:
You can init these in one shot like this:
```
const(
  rows = 20
  columns = 2
...
...
)
```

Same way you're doing it in `dataset` package. This is just a pure `Go` style comment 😇